plugins/uefi-dbx/meson.build

```diff
@@ -7,15 +9,28 @@ plugin_builtin_uefi_dbx = static_library('fu_plugin_uefi_dbx',
     'fu-uefi-dbx-plugin.c',
     'fu-uefi-dbx-common.c',
     'fu-uefi-dbx-device.c',
+    'fu-uefi-dbx-snapd-notifier.c',
   ],
   include_directories: plugin_incdirs,
   link_with: plugin_libs,
   c_args: cargs,
-  dependencies: plugin_deps,
+  dependencies: [
+    plugin_deps,
+    libcurl,
+  ],
 )
 plugin_builtins += plugin_builtin_uefi_dbx
 
 if get_option('tests')
+  install_data(['tests/dbx-d719b2cb-3d3a-4596-a3bc-dad00e67656f'],
+    install_dir: join_paths(installed_test_datadir, 'tests'))
+  install_data(['tests/KEK-8be4df61-93ca-11d2-aa0d-00e098032b8c'],
+    install_dir: join_paths(installed_test_datadir, 'tests'))
```

Comment:
I'm not a huge fan of adding binary tests blobs into the repo since the whole libxz supply chain issue. Can we build these files using a little python program generator instead?

Reply:
I'm not sure that's feasible. The content of dbx and KEK were pulled directly from efivars where I previously enrolled locally generated keys. For some reason the parser wasn't happy with pristine esl neither with self signed ones which I generated with efitools. So at best, if I somehow figure out why the files weren't parsed successfully, the tests will require efitools and some scripting to generate keys on the fly.